app/_hub/kong-inc/openid-connect/index.md

```diff
@@ -129,7 +129,7 @@ kong_version_compatibility:
       compatible:
     enterprise_edition:
       compatible:
-        - 2.4.x
+        - 2.6.x
```

Comment:
@falondarville I don't see the version compatibility show up in the preview, but I assume this might be because 2.6 is not released yet so it display anything for unknown versions.
![Screen Shot 2021-10-11 at 3 46 33 PM](https://user-images.githubusercontent.com/8764246/136847734-6068d24f-a55b-4fd5-b7a8-b0ae8764d453.png)